### PR TITLE
Map cb servervar

### PR DESCRIPTION
**Pull request by sam016**
Completed adding the OrderedMap for the structs along with their basic unit tests

1. OrderedCallbacks
2. OrderedEncodings
3. OrderedExamples
4. OrderedHeaders
5. OrderedLinks
6. OrderedMediaTypes
7. OrderedResponses
8. OrderedStrings
9. OrderedParameters
10. OrderedRequestBodies
11. OrderedResponses
12. OrderedSchemas
13. OrderedSecuritySchemes

These structs are created with `Ordered` prefix, and the old ones are also renamed to follow this pattern.

In the upcoming PRs, adding marshal and unmarshal to these new `Ordered...` structs

### CODE REVIEW

**Comment by sam016** [comment]
@casualjim missed adding you as a reviewer.

File: server_easyjson.go [hunk 1]
```diff
@@ -75,19 +134,16 @@ func easyjson22b57fa5DecodeGithubComGoOpenapiSpec3(in *jlexer.Lexer, out *Server
...
 	out.RawByte('{')
 	first := true
 	_ = first
-	if len(in.Enum) != 0 {
-		const prefix string = ",\"enum\":"
-		if first {
-			first = false
-			out.RawString(prefix[1:])
+	{
+		const prefix string = ",\"Enum\":"
```
Comment (casualjim) [inline server_easyjson.go]: The casing has changed, is that on purpose?

Comment (sam016) [reply server_easyjson.go]: No, it wasn't on purpose. I had to generate the easyjson file again once the structs in `server.go` got updated. And, I didn't use the `-lower_camel_case` argument while generating the easyjson file. That's why they were in CamelCase. Again, it's my fault also that I didn't check the difference of the generated files assuming they would be correct. Will be fixing it in 12 hrs.

**Review by casualjim — APPROVED** [review]

**Comment by casualjim** [comment]
excellent work so far!

File: server_easyjson.go (hunk 1 above)
Comment (sam016) [reply server_easyjson.go]: Generated the server and contact easyjson files correctly. And, also found your `go:generate` commands in `docs.go`. I wish I had found them earilier.

**Pull request merged**